deploy_carvel/.gitignore

```diff
@@ -0,0 +1,12 @@
+# package_repository/packages/{pkg}/ contains specific SHAs of images
```

Comment:
This would be good to review and verify what we do and don't want to keep in repo as artifacts.